kanidm_tools/src/cli/account.rs

```diff
@@ -310,12 +332,32 @@ impl AccountOpt {
             }
             AccountOpt::Delete(aopt) => {
                 let client = aopt.copt.to_client().await;
-                if let Err(e) = client
+                let mut modmessage = AccountChangeMessage {
```

Comment:
Is this adding the json mode?

Reply:
Yep, supports both text and JSON mode, depending on what the user's set